homework_4/routes/cohorts.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
+const database = require('../db/client');
+const router = express.Router();
+
+router.route('/').get((req, res) => {
+  database('cohorts')
+    //.orderBy('timestamp', 'desc')
```

Comment:
It would've been a good idea to order by timestamp so that the newest post is at the top - especially since you had seed data I had to scroll down to find the post I created. 
Looks like you didn't create a column for `created_at`. Reading through their doc, you can include this in the migration script `table.timestamp('created_at').defaultTo(knex.fn.now());`
https://knexjs.org/guide/schema-builder.html#timestamp